code/widget-showcase/src/examples/table.rs

```diff
@@ -0,0 +1,29 @@
+use ratatui::{prelude::*, widgets::*};
+
+pub fn render(frame: &mut Frame) {
+    let rows = [
+        Row::new(["1".bold(), "John".into(), "john.doe@mail.org".dim()]),
+        Row::new(["2".bold(), "Joe".into(), "joe.dane@mail.org".dim()]),
+        Row::new(["3".bold(), "Jane".into(), "jane.doe@mail.org".dim()]),
+        Row::new(["4".bold(), "John S.".into(), "john.smith@mail.org".dim()]),
+    ];
+
+    let table = Table::new(
+        rows,
+        [Constraint::Min(8), Constraint::Min(15), Constraint::Min(20)],
```

Comment:
Because we know the exact size of the image that we're displaying here, these lengths can be exact. 

Maybe make the ID column narrower.
To fill the space better you could make the user names lower case, and add a Name column.